CS3219-AY2425S1/cs3219-ay2425s1-project-g11
# Add hot reloading with docker and fix Husky

This PR aims to add hot reloading with docker

### How to test
- do `docker compose up --build`
- Then, go to any of the services (make sure that peerprep-fe is compiled if you want to test for that)
- Make a change and test if the service hot reloads
- Try a mock `git commit` to test if husky works
- Also, try the production build using `docker compose -f docker-compose.prod.yml up --build`

Closes [#43](https://github.com/CS3219-AY2425S1/cs3219-ay2425s1-project-g11/issues/43)
